tests/test-st2-invalid-auth.js

```diff
@@ -0,0 +1,154 @@
+/*
+ Licensed to the StackStorm, Inc ('StackStorm') under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*jshint quotmark:false*/
+/*jshint -W030*/
+/*global describe, it, before, after*/
+'use strict';
+
+var chai = require("chai"),
+  expect = chai.expect,
+  chaiString = require("chai-string"),
+  sinon = require('sinon'),
+  sinonChai = require('sinon-chai'),
+  mockedEnv = require('mocked-env'),
+  Robot = require("hubot/src/robot"),
+  Logger = require('./dummy-logger.js');
+
+chai.use(sinonChai);
+chai.use(chaiString);
+
+global.process.exit = sinon.spy();
+
+describe("invalid st2 credential configuration", function() {
```

Comment:
With these tests in hubot-stackstorm do we still need e2e tests located https://github.com/StackStorm/st2tests/pull/170?

Reply:
No, we'll close that PR once this PR is merged.